src/engagement_db_to_analysis/engagement_db_to_analysis.py

```diff
@@ -89,10 +93,48 @@ def get_project_messages_from_engagement_db(dataset_configurations, engagement_d
 
     return engagement_db_dataset_messages_map
 
-#TODO: Filter Messages
-def filter_messages():
-    return None
+def _convert_messages_to_traced_data(user, messages_map):
+    #TODO: add docsting
+
+    data = []
```

Comment:
Can we call this what it is e.g. messages, or messages_td?

Reply:
updated thanks!